### PR TITLE
[IBMCloud] auto OCP-73970 OCP-74704 with the existed step

### PR DESCRIPTION
**Pull request by MayXuQQ**
- [OCPQE-22452 automate case OCP-73970 - [IPI-on-IBMCloud] Install, reboot and destroy a cluster](https://issues.redhat.com/browse/OCPQE-22452) with [cucushift-installer-check-reboot-nodes](https://steps.ci.openshift.org/reference/cucushift-installer-check-reboot-nodes)

- auto [OCP-74704 - [IPI-on-IBMCloud] Install using ed25519 / ecdsa key](https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-74704) with [cucushift-installer-check-sshkey](https://steps.ci.openshift.org/reference/cucushift-installer-check-sshkey)

### CODE REVIEW

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.17-installer-rehearse-ibmcloud

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml [hunk 1]
```diff
@@ -2113,6 +2113,7 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe
     env:
+      ENABLE_REBOOT_CHECK: "true"
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:2116]: Suggest to run the test case in `openshift-verification-tests-master__installation-nightly-4.1N.yaml` like other platforms.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml [hunk 2]
```diff
@@ -263,6 +263,22 @@ tests:
...
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: ibmcloud-ipi-private-reboot-nodes-f28
+  cron: 44 3 12 * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    env:
+      ENABLE_REBOOT_CHECK: "true"
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi-private-with-ep-cos
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml:274]: suggest to rename job name to reflect the workflow `with-ep-cos`

Comment (MayXuQQ) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml:274]: new a workflow for the reboot, do not use the existed workflow?

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.17-ibmcloud-ipi-private-private-with-ep-cos-reboot-nodes-f28

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.17-ibmcloud-ipi-private-sshkey-f28

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml [hunk 3]
```diff
@@ -263,6 +263,22 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: ibmcloud-ipi-private-private-with-ep-cos-reboot-nodes-f28
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml:266]:
```suggestion
- as: ibmcloud-ipi-private-custom-cos-endpoint-reboot-nodes-f28
```

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.17-ibmcloud-ipi-private-sshkey-f28

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.17-ibmcloud-ipi-private-private-with-ep-cos-reboot-nodes-f28

**Comment by MayXuQQ** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.17-ibmcloud-ipi-private-custom-cos-endpoint-reboot-nodes-f28

**Pull request merged**